paket.dependencies

```diff
@@ -1,5 +1,7 @@
 framework: auto-detect
```

Comment:
With framework on auto it's adding a lot to the project file, can we specify the correct frameworks (net 4.5 and 4.6.2)

Reply:
Okay. Will check that out.